examples/reminders/lib/base/models/counter/increment_operation.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:rx_bloc_list/models.dart';
+
+enum IncrementOperation {
+  /// The operation, which increments the incomplete counter and decrements
+  /// the complete counter
+  incrementIncompleteDecrementComplete,
+
+  /// The operation, which decrements the incomplete counter and increments
+  /// the complete counter
+  decrementIncompleteIncrementComplete,
+}
+
+/// This operation should be in the reminders list
+enum CounterOperation {
+  /// The operation, which just increments the incomplete counter
+  create,
+
+  /// The operation, which decrements the incomplete or complete counter
+  delete,
+
+  /// The operation, which decrements incomplete and increments complete or
+  /// increments incomplete and decrements complete or
+  /// does not change counter
+  update,
+}
+
+class ManagedListCounterOperation{
```

Comment:
Please pass the generic annotation to the ManagedList<E>

Reply:
I have passed a generic annotation to the ManagedList in this class